dotenv/README.md

```diff
@@ -0,0 +1,171 @@
+# Dotenv handling for deno.
+
+## Usage
+
+Setup a `.env` file in the root of your project.
+
+```sh
+# .env
+GREETING=hello world
+```
+
+Then import the configuration using the `config` function.
+
+```ts
+// app.ts
+import { config } from "https://deno.land/std@$STD_VERSION/dotenv/mod.ts";
+
+console.log(config());
+```
+
+Then run your app.
+
+```
+> deno app.ts
```

Comment:
This doesn't seem correct - missing `run` subcommand and it should require `--allow-read` flag